# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/text/test_lineboundary.html
# Commit: 7aa2279f941c
# Full Hash: 7aa2279f941cf9c5f5fdf4d786604ac7dca053ea
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2019-09-26 04:07:05
# Regressor Bug: 1177014
# File Overlap Count: 1
# Description:
#   Bug 1177014: Modify TransformOffset to correctly report bullet offsets, despite incorrect return from PeekOffset. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46490
# ==============================================================================

diff -r e79b84e42250 -r 7aa2279f941c accessible/tests/mochitest/text/test_lineboundary.html
--- a/accessible/tests/mochitest/text/test_lineboundary.html	Wed Sep 25 20:38:55 2019 +0000
+++ b/accessible/tests/mochitest/text/test_lineboundary.html	Wed Sep 25 02:10:08 2019 +0000
@@ -164,6 +164,13 @@
       testTextAtOffset(["ht_5" ], BOUNDARY_LINE_START,
                        [ [ 0, 0, kEmbedChar, 0, 1 ] ]);
 
+      // ////////////////////////////////////////////////////////////////////////
+      // Block followed by list
+
+      testTextAtOffset([ "block_then_ul" ], BOUNDARY_LINE_START,
+                       [ [ 0, 0, kEmbedChar, 0, 1 ],
+                         [ 1, 1, kEmbedChar, 1, 2 ] ]);
+
       SimpleTest.finish();
     }
 
@@ -258,6 +265,11 @@
     <center><input type="TEXT"><input value="Button" type="SUBMIT"></center>
     Line 3
   </div>
+
+  <div id="block_then_ul">
+    <p>Block</p>
+    <ul><li>Li</li></ul>
+  </div>
   </body>
 </body>
 </html>
